index.html

```diff
@@ -1843,8 +1843,9 @@ <h4 id="maturity-levels">6.1.2 Maturity Levels</h4>
               <dd>A Working Group may make editorial or other minor corrections to a Recommendation, and produce a new
               version which W3C publishes as a revised edition of the Recommendation.</dd>
              <dt>An Amended Recommendation</dt>
-              <dd>Where there is no Working Group chartered to maintain a Recommendation W3C may produce a new
-              version which W3C publishes as an Amended version of the Recommendation.</dd>
+              <dd>An <dfn id="rec-amended">Amended Recommendation</dfn> is a Recommendation that is amended to include <a href="#correction-classes">substantive 
+              changes that do not add new features</a>, and is produced by the W3C at a time when the Recommendation does not fit 
```

Comment:
I would reverse the order of the sub-clauses, along the line sof

> ... is produced by W3C w/out WG. An Amended Rec _must_ not include new features.

Reply:
seems plausible, draft away, and I'll comment if needed